examples/studio/assistant/user_defined_plans.py

```diff
@@ -0,0 +1,44 @@
+from ai21 import AI21Client
+from pydantic import BaseModel
+
+TIMEOUT = 20
+
+CODE_STR = """def create_table_and_summary(user_message: str):\n    company_ticker = step(\n
```

Comment:
I see this is not used. What do you think of moving it to its own example file?

Reply:
I think I'll remove it because passing a function is actually more comfortable